<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tetris</class>
 <widget class="QMainWindow" name="Tetris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_scorel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 127);
font: 75 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_scorel_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color: rgb(255, 255, 127);
font: 75 14pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_G">
    <property name="title">
     <string>游戏(&amp;G)</string>
    </property>
    <addaction name="action_start"/>
    <addaction name="action_pause"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>设置(&amp;S)</string>
    </property>
    <addaction name="action_easy"/>
    <addaction name="action_middle"/>
    <addaction name="action_difficult"/>
    <addaction name="separator"/>
    <addaction name="action_classic"/>
    <addaction name="action_magic"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_G"/>
   <addaction name="menu_S"/>
   <addaction name="menu_H"/>
  </widget>
  <action name="action_start">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>游戏说明</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于Tetris</string>
   </property>
  </action>
  <action name="action_easy">
   <property name="text">
    <string>容易</string>
   </property>
  </action>
  <action name="action_middle">
   <property name="text">
    <string>中等</string>
   </property>
  </action>
  <action name="action_difficult">
   <property name="text">
    <string>高级</string>
   </property>
  </action>
  <action name="action_classic">
   <property name="text">
    <string>经典模式</string>
   </property>
  </action>
  <action name="action_magic">
   <property name="text">
    <string>奇幻模式</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
